ui/src/app/views/workspace/component.ts

```diff
@@ -152,7 +148,20 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
                       this.cohortsLoading = false;
                       this.cohortsError = true;
                     });
-          },
+            this.errorHandlingService.retryApi(this.workspacesService
+              .getNoteBookList(this.wsNamespace, this.wsId))
+                .subscribe(
+                  FileList => {
```

Comment:
This is a parameter name, please use lower camel casing. If you want to type it as a FileList, use `(fileList: Array<FileDetail>) => ...`